### ISSUE TITLE
PHP fatal on install (autoloader not loading feeds classes)

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/feeds#12]
For some reason the autoloader isn't loading the feeds classes on my site. I've tried changing PHP versions to no avail. When I attempt to enable the Feeds module, at the moment I save the modules page I get the following error:

```
Error: Class 'FeedsConfigurable' not found in feeds_importer() (line 1066 of backdrop/modules/contrib/feeds/feeds.module).
```

When I add a manual call to load the file, this class is found, but the next one is not:
```php
  module_load_include('inc', 'feeds', 'includes/FeedsConfigurable');
```

I then get the following error:
```
Fatal error: Class name must be a valid object or a string in backdrop/modules/contrib/feeds/includes/FeedsConfigurable.inc on line 64
```

I was able to get around the second PHP fatal by wrapping the return of the instance with a check for class:
```php
    if (!empty($class)) {
      if (!isset($instances[$class][$id])) {
        $instances[$class][$id] = new $class($id);
      }
      return $instances[$class][$id];
    }
```

But I'm still not sure why any of this was necessary...

**Comment by laryn** [comment]
I just installed `feeds` and did not get the first error after enabling the module, but got the second error when trying to add a new Feed importer. I tried your fix above (`if (!empty($class)) { ...` and then tried to create an importer. That gave me this error:

`Error: Call to a member function configFormValidate() on null in feeds_ui_create_form_validate() (line 193 of /path/modules/feeds/feeds_ui/feeds_ui.admin.inc). `

The strange thing is, I believe I've installed and used Feeds on a different site without this issue.
